From 1d510e6049e7c7cb7ed4e87780b2c1a997d47b64 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 17 Nov 2012 19:36:39 +0100
Subject: [PATCH] attr: Fix static build installation

When doing a static build the .lai file is not created, so do not try
to install it.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 include/buildmacros |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/buildmacros b/include/buildmacros
index ab89182..d81e62b 100644
--- a/include/buildmacros
+++ b/include/buildmacros
@@ -98,8 +98,7 @@ endif
 
 INSTALL_LTLIB_STATIC = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR)
+	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR);
 
 INSTALL_MAN = \
 	@for d in $(MAN_PAGES); do \
-- 
1.7.9.5

